docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
Assuming you have docker version 17.05+ you should be able to use multistage builds